client/src/app/shared/menu/top-menu-dropdown.component.ts

```diff
@@ -85,6 +105,31 @@ export class TopMenuDropdownComponent implements OnInit, OnDestroy {
     this.openedOnHover = false
   }
 
+  @HostListener('window:resize')
+  onWindowResize () {
+    this.isInSmallView = !!this.screen.isInSmallView()
+  }
```

Comment:
Why not use a getter? This would prevent having to setup another listener, which is why we use a global service like ScreenService…

```
get isInSmallView () {
  return this.screen.isInSmallView()
}
```

Reply:
oh ! thanks I never thought about getter ! it's magic :)